recipe/run_test.bat

```diff
@@ -0,0 +1,4 @@
+:: Test boost::iostreams zlib filter support
```

Comment:
Same as with `bash`, it might be worth echoing command lines run (and any output) to make sure things are running ok.

Reply:
```suggestion
:: Test boost::iostreams zlib filter support

@echo on
```